docs/reference/configuration.rst

```diff
@@ -990,6 +990,17 @@ Default value of the ``cache`` attribute of plugins. Should plugins be cached by
     If you disable the plugin cache be sure to restart the server and clear the cache afterwards.
 
 
+..  setting:: CMS_CACHE_LIMIT_TTL_CLASS
+
+CMS_CACHE_LIMIT_TTL_CLASS
+================
```

Comment:
I think this line needs to match the length of the title to avoid a syntax error.

Reply:
Fixed.